base/linalg/uniformscaling.jl

```diff
@@ -171,6 +171,15 @@ function isapprox(J1::UniformScaling{T}, J2::UniformScaling{S};
             rtol::Real=Base.rtoldefault(T,S), atol::Real=0, nans::Bool=false) where {T<:Number,S<:Number}
     isapprox(J1.λ, J2.λ, rtol=rtol, atol=atol, nans=nans)
 end
+function isapprox(J::UniformScaling,A::AbstractMatrix;kwargs...)
+    n = checksquare(A)
+    D = diagm(J.λ*ones(typeof(J.λ),n))
+    return isapprox(D,A;kwargs...)
+end
+function isapprox(A::AbstractMatrix,J::UniformScaling;
+                  kwargs...)
+    return isapprox(J,A;kwargs...)
+end
```

Comment:
Stylistically, this function should normally be written on a single line as in my suggestion.